articles/synapse_docker_registry.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Synapse Docker Registry
+layout: article
+excerpt: The basics of using the synapse docker registry
```

Comment:
Should be a little more extensive and summarize why someone would care and what they will learn.